README.md

```diff
@@ -622,7 +622,7 @@ Other Style Guides
     });
     ```
 
-  - [8.2](#8.2) <a name='8.2'></a> If the function body consists of a single expression, feel free to omit the braces and use the implicit return. Otherwise use a `return` statement.
+  - [8.2](#8.2) <a name='8.2'></a> If the function body consists of a single expression, feel free to omit the braces and use the implicit return. Otherwise add the parentheses, braces, and use a `return` statement.
```

Comment:
What we've agreed to:

* If you're using braces then the parentheses are always required.
* If it fits in a single line then feel free to omit anything (parentheses, braces, return).